app/scripts/views/settings/devices.js

```diff
@@ -75,7 +50,13 @@ define(function (require, exports, module) {
 
     _formatDevicesList: function (devices) {
       return _.map(devices, function (device) {
-        device.lastAccessTime = $.timeago(Number(device.lastAccessTime));
+        if (device.lastAccessTimeFormatted) {
+          device.lastAccessTime = Strings.interpolate(
+            t('Last active: %(translatedTimeAgo)s'), { translatedTimeAgo: device.lastAccessTimeFormatted });
```

Comment:
Copying over [this comment I made on the fxa-js-client PR](https://github.com/mozilla/fxa-js-client/pull/212#issuecomment-239112038):

> @vladikoff - touching 3 repos to display the "time ago" seems pretty heavy handed. I didn't realize it'd be this complex. Couldn't we just use the singular/plural functionality built into jquery.timeago, explain to the l10n team why we have split up the strings that way, and away we go?